server/src/employee/employee.module.ts

```diff
@@ -6,11 +6,12 @@ import { EmployeeService } from './employee.service';
 import { Manager } from 'src/manager/manager.entity';
 import { LocalStrategy } from 'src/authentication/local.strategy';
 import { AuthenticationService } from 'src/authentication/authentication.service';
+import { ManagerService } from 'src/manager/manager.service';
 
 @Module({
   imports: [TypeOrmModule.forFeature([Employee, Manager])],
-  providers: [EmployeeService, LocalStrategy, AuthenticationService],
+  providers: [EmployeeService, LocalStrategy, AuthenticationService, ManagerService],
   controllers: [EmployeeController],
-  exports: [EmployeeModule]
+  exports: [EmployeeModule, EmployeeService]
```

Comment:
```suggestion
  exports: [EmployeeService]
```

pas besoin de EmployeeModule dans `exports`